python/pyspark/sql/connect/README.md

```diff
@@ -9,22 +9,34 @@ of Spark. To enable it, you only need to activate the driver plugin for Spark Co
 
 ## Build
 
-1. Build Spark as usual per the documentation.
-
-2. Build and package the Spark Connect package
-
-   ```bash
-   ./build/mvn -Phive package
-   ```
+```bash
+./build/mvn -Phive clean package
+```
 
-   or
+or
 
-   ```bash
-   ./build/sbt -Phive package
-   ```
+```bash
+./build/sbt -Phive clean package
+```
    
 ## Run Spark Shell
 
+To run Spark Connect you locally built:
+
+```bash
+# Scala shell
+./bin/spark-shell \
+  --jars `ls connector/connect/target/**/spark-connect*SNAPSHOT.jar | paste -sd ',' -` \
+  --conf spark.plugins=org.apache.spark.sql.connect.SparkConnectPlugin
+
+# PySpark shell
+./bin/pyspark \
+  --jars `ls connector/connect/target/**/spark-connect*SNAPSHOT.jar | paste -sd ',' -` \
+  --conf spark.plugins=org.apache.spark.sql.connect.SparkConnectPlugin
+```
+
+To use the release version of Spark Connect:
+
 ```bash
 ./bin/spark-shell \
   --packages org.apache.spark:spark-connect_2.12:3.4.0 \
```

Comment:
Is this the preferred way of doing it?

Reply:
Yup, the user-facing documentation will only have this alone.